mjsimulateui/array_safety.h

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 DeepMind Technologies Limited
```

Comment:
I have not come up with a good way to avoid copying this file from the `sample` directory. If it were a part of the MuJoCo headers, it would not need to be copied.